.github/workflows/trigger_nightly.yml

```diff
@@ -18,6 +18,7 @@ on:
           - torchrec
           - tensorrt
           - data
+          - FBGEMM
```

Comment:
probably need to lower-case all mentions of fbgemm

Reply:
Thank you, fixed!

https://github.com/pytorch/test-infra/pull/4575/commits/6f501a9f60cd1e74a563f0d95561f0f2cfd3eae8